src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/VmrCloneManager.cs

```diff
@@ -76,8 +76,7 @@ public async Task<ILocalGitRepo> PrepareVmrAsync(
             cancellationToken);
 
         _vmrInfo.VmrPath = vmr.Path;
-        await _dependencyTracker.InitializeSourceMappings();
-        _sourceManifest.Refresh(_vmrInfo.SourceManifestPath);
+        await _dependencyTracker.RefreshMetadata();
```

Comment:
Did you look in other usages of `InitializeSourceMappings` whether they can also change to this?

Reply:
I think we will need to be able to override the `sourceMappingPath` too